docs/UserGuide.md

```diff
@@ -89,30 +89,67 @@ This section provides support for the commands required to perform desired actio
 
 **:information_source: Note for command input format:**<br>
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `[command word] n/<name> q/<source of stock>`,
-  `[command word] q/<source of stock> n/<name>` is also acceptable.
-* Duplicate parameters are not allowed.
-  e.g. `[command word] n/<name> n/<name>` is an invalid command format and will show an error message.
+* Words enclosed in `<>` are the input parameters to be supplied by the user. <br>
+  For example, in `n/<name>`, `name` is a parameter which can be used as `n/Pork Belly`.
+
+* Prefixes not enclosed with anything is compulsory, must be provided by the user, and duplicate is not allowed. <br>
+  For example, in `list lt/<list type>`, the `lt/<list type>` must be provided and no duplicate is allowed. <br>
+  `list` and `list lt/all lt/bookmark` are invalid command formats, but `list lt/all` is valid.
+
+* Prefixes enclosed with `[]` is optional, can be omitted by the user, and duplicate is not allowed. <br>
+  For example, in `update sn/<serial number> [n/<name]`, the `n/<name>` can be omitted. <br>
+  `update sn/Fairprice1 n/Apple` and `update sn/Fairprice1` are both valid command formats, <br>
+  but `update sn/Fairprice1 n/Apple n/Banana` is not a valid command format.
+
+* Compulsory prefixes with `...` trailing after them can be used more than one time in one command. <br>
+  For example, in `delete sn/<serial number>...`, the `sn/<serial number>` must be provided and duplicates are allowed. <br>
+  `delete sn/Fairprice1` and `delete sn/Fairprice1 sn/Ntuc1` are both valid command formats.
   
+* Multiple prefixes combined with `|` means only one of them can be provided at a single command. <br>
+  For example, in `update sn/<serial number> [iq/<increment value> | nq/<new quantity>]`, at most one of `iq/<increment value>`
+  or `nq/<new quantity>` may be provided. <br>
+  `update sn/Fairprice iq/+10 nq/100` is an invalid command format, but `update sn/Fairprice1 iq/+10`, 
```

Comment:
Is there supposed to be a 1 behind the fairprice here?

Reply:
Yes. Thanks for mentioning!